src/core_plugins/kibana/public/dev_tools/hacks/disable.js

```diff
@@ -0,0 +1,16 @@
+import chrome from 'ui/chrome';
+import modules from 'ui/modules';
+
+modules.get('kibana').run(function ($rootScope, $location) {
```

Comment:
This doesn't feel like the right way to do this, what other methods were ruled out? A route resolve feels like a more appropriate way to add the precondition, and it should probably be 1) paired with a `notify.warning()` and 2) `kbnUrl.redirect()`